integration_test/third_party_apps_data/applications/hbase/debian_ubuntu/install

```diff
@@ -2,7 +2,12 @@ set -e
 set -o pipefail
 
 sudo apt-get update
-sudo apt-get install -y default-jre wget
+source /etc/os-release
+if [ "${VERSION_CODENAME}" = "lunar" ]; then
+  sudo apt-get install -y openjdk-11-jre wget
```

Comment:
Is this because `default-jre` on Ubuntu 23.04 pulls in `openjdk-17-jre` (https://www.ubuntuupdates.org/package/core/lunar/main/base/default-jre) instead? What's wrong with HBase on JDK 17? Worth a comment, at least.